app/main/views/services.py

```diff
@@ -42,16 +42,25 @@ def list_services():
     supplier_id = get_int_or_400(request.args, 'supplier_id')
 
     if request.args.get('framework'):
-        services = Service.query.has_frameworks(*[
-            slug.strip() for slug in request.args['framework'].split(',')
-        ])
+        frameworks = [slug.strip() for slug in request.args['framework'].split(',')]
     else:
-        services = Service.query.framework_is_live()
+        frameworks = None
 
     if request.args.get('status'):
-        services = services.has_statuses(*[
-            status.strip() for status in request.args['status'].split(',')
-        ])
+        statuses = [status.strip() for status in request.args['status'].split(',')]
+    else:
+        statuses = None
+
+    try:
+        services = filter_services(
+            framework_slugs=frameworks,
+            statuses=statuses,
+            lot_slug=request.args.get('lot_slug', None),
```

Comment:
We use `framework` when passing framework slugs as query arguments, should we just use `lot` for lots?